modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml

```diff
@@ -109,11 +167,13 @@ paths:
             format: int32
             default: 20
         - name: sortProperty
+          description: Selected property used in ordering the search results.
```

Comment:
I'd suggest "Property to use to sort the search results."
If we know, we should describe what are the possible values, and what the behavior is when this parameter is not indicated.